lightwood/mixers/helpers/default_net.py

```diff
@@ -15,6 +16,7 @@ def __init__(self, dynamic_parameters,
                      output_size=None,
                      nr_outputs=None,
                      shape=None,
+                     target_params=3e5,
```

Comment:
Should probably measure GPU free memory size and use that, also rename to `max params` since we aren't "targeting this" we are "targeting being bellow this"

Reply:
Hm, if we do it based on free GPU memory, doesn't it get tricky when we have more than one concurrent predictor? Will rename to `max_params`.